### DISCUSSION TITLE
How to bring up suggestions again

### DISCUSSION THREAD

**Discussion by feelingsonice** [zed-industries/zed#7013] [Feature Requests]
Attached a video of what I'm talking about. In the video, you can see that as I was typing `size.` suggestions came up as I was typing. However, after I backtracked and deleted back to `size.`, deleting and then typing the `.` would not trigger the suggestion boxes again. Is there a hotkey for this?

https://github.com/zed-industries/zed/assets/42384776/3766ba34-029d-4eac-9869-b59679c15787

**Comment by Moshyfawn** [comment]
https://github.com/zed-industries/zed/discussions/8233#discussioncomment-8563572
